packages/nameguard-sdk/src/index.ts

```diff
@@ -554,7 +583,7 @@ export class NameGuard {
    * Inspects the name associated with a namehash.
    *
    * NameGuard will attempt to resolve the name associated with the namehash through the ENS Subgraph.
-   * If this resolution succeeds then NameGuard will generate and return a `NameGuardReport` for the name.
+   * If this resolution succeeds then NameGuard will return a `NameGuardReport` for the name. If the resolved `name` is uninspected then NameGuard will return `UninspectedNameGuardReport`.
```

Comment:
@djstrong I think it will be nice to update this to more formally refer to the new `inspected` field rather than the abstract idea of being inspected / uninspected.